brightwind/load/station.py

```diff
@@ -1071,9 +1071,10 @@ def __get_names(self, measurement_type_id=None):
         :param measurement_type_id: The measurement_type_id to filter for the names.
         :type measurement_type_id:  str or None
         :return:                    The list of names.
-        :rtype:                     list(str)
+        :rtype:                     str | list[str]
```

Comment:
I don't get why you made this update. The output of the function before and after your fix is a list(str), it will never happen that it is a str as `names` is defined as a list on function e.g. `names = []`. Did you mean to update the input 'measurement_type_id' type?